src/AutoRest.CSharp/Mgmt/AutoRest/CodeModelValidator.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Utilities;
+
+namespace AutoRest.CSharp.Mgmt.AutoRest
+{
+    internal class CodeModelValidator
+    {
+        public static void Validate(CodeModel model) {
+            foreach (var group in model.OperationGroups)
+            {
+                if (string.IsNullOrEmpty(group.Key)) {
+                    ErrorHelpers.ThrowError("Malformed 'operationId' detected. Check swagger definition and make sure every 'operatoinId' follows form of 'Noun_Verb'." +
```

Comment:
`@"`?

Reply:
Thanks. I didn't realize the c# multiple line directive.